### ISSUE TITLE
Reboot workers manually via device control causes exception due to missing function

### ISSUE THREAD

**Issue by sn0opy** [Map-A-Droid/MAD#716]
At some point, a certain function to force a restart of a worker has been removed causing an exception. The location where it happens:

```
 File "MAD/mapadroid/madmin/routes/control.py", line 305, in restart_phone
    self._ws_server.force_disconnect(origin)
AttributeError: 'WebsocketServer' object has no attribute 'force_disconnect'
```

**Comment by crhbetz** [comment]
^ it got killed in #705 apparently.

**Comment by Expl0dingBanana** [comment]
Working on getting this fixed ATM

**Comment by Grennith** [comment]
Should be fixed with 997253ba1366eb2b9b8cf2aee23886c72e498c36

**Comment by Grennith** [comment]
Any feedback in that regard? :)
